db/migrate/20220809133409_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Good job adding reference to the foreign key but according to the ERD diagram below, the user table should be referenced with an alias of `author`. Please kindly implement this.

![](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png?raw=true)

Reply:
Hi @AnselemOdims Thank you for your kind response. I have one question, should I recreate the migrations or just modify the existing ones?